src/pages/using-spark/components/button.mdx

```diff
@@ -94,18 +55,18 @@ Represents the main call to action on the page. Use no more than two Primary But
   titleAttr="Primary Button Example"
 />
 
-### Secondary Button
+- Use in areas of high importance.
+- Use one Primary (Filled) button per screen to help users understand the action hierarchy of the page and to avoid any confusion.
+- For actions that help users navigate to the next step in a process.
+
+**Exception**
+
+For specific situations where users can choose between two journeys, two Primary (Filled) buttons may exist side by side.
```

Comment:
We've never used the word "journeys" before so it's a metaphor that comes out of nowhere. 

I would simply and say "between two options"